common/views/Home.jsx

```diff
@@ -1,85 +1,114 @@
-var React = require('react'),
-    Transmit = require('react-transmit'),
-    mui = require('material-ui');
+import React from 'react';
+import Transmit from 'react-transmit';
+import { Avatar, Card, CardTitle, CardActions, ListItem, ListDivider,
+  RaisedButton } from 'material-ui';
+import MoreVertIcon from 'material-ui/lib/svg-icons/navigation/more-vert';
+import ForwardIcon from 'material-ui/lib/svg-icons/navigation/arrow-forward';
+import WelcomeSection from './WelcomeSection.jsx';
+import RateSection from './RateSection.jsx';
+import pptColors from '../styles/color';
+import pptSpacing from '../styles/spacing';
 
-const Spacing = mui.Styles.Spacing;
+class Home extends React.Component {
+
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      wasVisited: true
+    };
+  }
+
+  componentDidMount() {
+    if (!(window.localStorage.getItem('wasVisited'))) {
+      window.localStorage.setItem('wasVisited', false)
+      this.setState({
+        wasVisited: false
+      });
+    }
+  }
+
+  onKnownTouchTap() {
+    window.localStorage.setItem('wasVisited', true)
+    React.findDOMNode(this.refs.welcomeSection).style.height = 0;
+  }
 
-var Home = React.createClass({
   getStyles() {
     return {
-      root: {paddingTop: Spacing.desktopKeylineIncrement},
-      noMarginTop: {marginTop: 0},
-      small: {color: 'rgba(255,255,255,0.6)', marginTop: 0},
-      noBothMargin: {marginTop: 0, marginBottom: 0},
-      button: {margin: '10px auto'}
+      root: {
+        width: '100%',
+        height: '100%',
+        paddingTop: pptSpacing.appBarHeight
```

Comment:
Maybe adding `8px` to `paddingLeft`, `paddingRight` may look closer to mockup.

<img width="560" alt="2015-08-02 11 08 44" src="https://cloud.githubusercontent.com/assets/108608/9026703/88630e7a-396b-11e5-9f90-aeb21e59ab33.png">